test/cases/fixtures_test.rb

```diff
@@ -370,9 +349,9 @@ def teardown
         t.integer "a" * max_identifier_length
       end
 
-      Company.connection.drop_table :companies, if_exists: true
-      Company.connection.exec_query("DROP SEQUENCE IF EXISTS companies_nonstd_seq")
-      Company.connection.create_table :companies, force: true do |t|
+      Company.lease_connection.drop_table :companies, if_exists: true
+      Company.lease_connection.exec_query("DROP SEQUENCE IF EXISTS companies_nonstd_seq CASCADE")
```

Comment:
This started failing with CRDB v24.1 with message:

> cannot drop sequence companies_nonstd_seq because other objects depend on it

It would be nice to see what are the depending objects. In PG there's this command:

```sql
CREATE OR REPLACE VIEW dependencies AS 
SELECT DISTINCT srcobj.oid AS src_oid
  , srcnsp.nspname AS src_schemaname
  , srcobj.relname AS src_objectname
  , tgtobj.oid AS dependent_viewoid
  , tgtnsp.nspname AS dependant_schemaname
  , tgtobj.relname AS dependant_objectname
FROM pg_class srcobj
  JOIN pg_depend srcdep ON srcobj.oid = srcdep.refobjid
  JOIN pg_depend tgtdep ON srcdep.objid = tgtdep.objid
  JOIN pg_class tgtobj ON tgtdep.refobjid = tgtobj.oid AND srcobj.oid <> tgtobj.oid
  LEFT JOIN pg_namespace srcnsp ON srcobj.relnamespace = srcnsp.oid
  LEFT JOIN pg_namespace tgtnsp ON tgtobj.relnamespace = tgtnsp.oid
WHERE tgtdep.deptype = 'i'::"char" AND tgtobj.relkind = 'v'::"char";
```

<sup>https://stackoverflow.com/a/48770535/6320039</sup>

But it doesn't work in CRDB. Any idea @rafiss ?

Reply:
crdb_internal.forward_dependencies is what you are looking for. add a `WHERE descriptor_name = 'companies_nonstd_seq'`; the object that is depended on by this sequence will be in the `dependedonby_id`

you could probably join this with backward_dependencies to get a more illustrative picture:
```
demo@127.0.0.1:26257/movr> SELECT                                                                                          
                        ->     b.descriptor_id AS dependent_id,                                                            
                        ->     f.descriptor_name AS depended_on_by,                                                        
                        ->     b.descriptor_name AS depedent_name                                                          
                        -> FROM                                                                                            
                        ->     crdb_internal.backward_dependencies AS b                                                    
                        -> JOIN                                                                                            
                        ->     crdb_internal.forward_dependencies AS f                                                     
                        -> ON                                                                                              
                        ->     f.dependedonby_id = b.descriptor_id                                                         
                        -> WHERE f.descriptor_name = 'companies_nonstd_seq';                                               
  dependent_id |    depended_on_by    | depedent_name
---------------+----------------------+----------------
           113 | companies_nonstd_seq | companies
(1 row)
```